### PR TITLE
Update all hapi libs to new module location

### PR DESCRIPTION
**Pull request by cameronmoreau**
All hapi modules were moved; non `@hapi/*` modules are no longer maintained

Looks like a previous commit or the `lab` upgrade dropped the coverage. This PR is a draft until CC is back at 100%

### CODE REVIEW

**Comment by johnbrett** [comment]
thanks for working on this @cameronmoreau - do you want me to review this or wait till you give the heads up?

**Comment by cameronmoreau** [comment]
@johnbrett np, back to 💯 so ready for review!

**Comment by johnbrett** [comment]
This is awesome work, no changes to tests 😍 always like to see that! Have left one question is all!

**Comment by cameronmoreau** [comment]
I'm probably just blind but I can't find the question you left. Do you mind linking to it @johnbrett ?

**Comment by johnbrett** [comment]
oh I think I had to hit "submit review" - does it show now ?

File: package.json [hunk 1]
```diff
@@ -22,21 +22,21 @@
...
-    "code": "^5.1.2",
-    "hapi": "^17.0.1",
-    "lab": "^17.0.0"
+    "@hapi/code": "^5.3.1",
+    "@hapi/hapi": "^17.9.0",
+    "@hapi/lab": "^19.0.1"
   },
   "scripts": {
-    "test": "lab -a code -t 100 -L",
-    "test-cov-html": "lab -a code -r html -o coverage.html"
```
[the remark this answers is not in the document]
Comment (cameronmoreau) [reply package.json]: Ahh, so after doing some research, I had a slight misunderstanding of what the "assert" flag did. I believed it was only used in the case that you'd like to swap assertion libraries for different reports.

Realized that the flag does a little more like reporting [incomplete assertions and verbosity](https://github.com/hapijs/lab#integration-with-an-assertion-library). I'll update with the appropriate `-a @hapi/code`. :+1:

**Comment by johnbrett** [comment]
@cameronmoreau merged, and published as v6.2.0!

Thanks for this update, much appreciated 👍

**Comment by taina0407** [comment]
@johnbrett could you please update 
`"peerDependencies": {
    "hapi": ">=17.x.x"
  }`
to 
`"peerDependencies": {
    "@hapi/hapi": ">=17.x.x"
  }`

This will avoid `warning " > hapi-auth-bearer-token@6.2.0" has unmet peer dependency "hapi@>=17.x.x"`

**Comment by cameronmoreau** [comment]
Whoops @taina0407, fixed and pr'd #185

**Comment by johnbrett** [comment]
Nice catch @taina0407, thanks for fix @cameronmoreau. Merged and published as v6.2.1

**Pull request merged**